src/main/java/it/gov/pagopa/pu/bff/service/AuthorizationService.java

```diff
@@ -47,4 +50,15 @@ public AccessTokenDTO postToken(String idToken) {
         null));
   }
 
+  public void validateAdminRole(Long organizationId, UserInfo loggedUser) {
+    boolean roleAdmin = loggedUser.getOrganizations().stream()
+      .filter(o->!CollectionUtils.isEmpty(o.getRoles()))
+      .anyMatch(o ->
+        organizationId.equals(o.getOrganizationId())
+        && o.getRoles().contains(ROLE_ADMIN));
```

Comment:
written ASIS, if the stream find the requested organization, but the user is not an admin, it will continue to search between its organizations (it will not have too many organizations, so it's not a problem, but it could be optimized
```suggestion
      boolean roleAdmin = loggedUser.getOrganizations().stream()
         .filter(o->organizationId.equals(o.getOrganizationId())
         .findFirst()
         .filter(o ->!CollectionUtils.isEmpty(o.getRoles()) && o.getRoles().contains(ROLE_ADMIN))
         .isPresent();
```